src/store/actions/taskActions.js

```diff
@@ -0,0 +1,14 @@
+import { ADD_TASK } from "src/store";
+import { createDate } from "src/utils/helpers/createDate";
+
+export const addTasks = (title) => {
+  return {
+    type: ADD_TASK,
+    payload: {
+      id: Date.now().toString(),
+      isCompleted: false,
+      title: title,
```

Comment:
Can we use [object-shorthand](https://eslint.org/docs/latest/rules/object-shorthand) ?  [Reference](https://github.com/azo-wtag/todo-app-react/pull/1#discussion_r1201471808)
```suggestion
       title,
```